gensim/matutils.py

```diff
@@ -532,6 +532,10 @@ def jaccard(vec1, vec2):
         return 1 - float(len(intersection)) / float(len(union))
 
 
+def jaccard_set(set1, set2):
+    return 1. - float(len(set1 & set2)) / float(len(set1 | set2))
```

Comment:
Will throw an exception if both inputs empty -- is that desired?

Missing docstring.